rules/libyang1.mk

```diff
@@ -0,0 +1,43 @@
+# libyang1
+
+LIBYANG1_VERSION_BASE = 1.0
+LIBYANG1_VERSION = $(LIBYANG1_VERSION_BASE).184
+LIBYANG1_SUBVERSION = 2
+LIBYANG1_FULLVERSION = $(LIBYANG1_VERSION)-$(LIBYANG1_SUBVERSION)
+
+export LIBYANG1_VERSION_BASE
+export LIBYANG1_VERSION
+export LIBYANG1_SUBVERSION
+export LIBYANG1_FULLVERSION
+
+LIBYANG1 = libyang1_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(LIBYANG1)_SRC_PATH = $(SRC_PATH)/libyang1
+SONIC_MAKE_DEBS += $(LIBYANG1)
+
+LIBYANG1_DEV = libyang-dev_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_DEV)))
+
+LIBYANG1_DBG = libyang1-dbgsym_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_DBG)))
+
+LIBYANG1_CPP = libyang-cpp1_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(LIBYANG1_CPP)_DEPENDS += $(LIBYANG1)
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP)))
+
+LIBYANG1_CPP_DEV = libyang-cpp-dev_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP_DEV)))
+
+LIBYANG1_CPP_DBG = libyang-cpp1-dbgsym_$(LIBYANG1_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(eval $(call add_derived_package,$(LIBYANG1),$(LIBYANG1_CPP_DBG)))
```

Comment:
The naming of .deb looks a little bit inconsistent. Seems like a mixture of "libyang-*", "libyang1-*", "libyang-*1".

Reply:
this is defined by debian packing name  https://packages.debian.org/source/bullseye/libyang

I think they may have some naming convention, i am not sure.